src/EES/Register/Register.js

```diff
@@ -126,28 +127,52 @@ const Register = () => {
       <div className="form-container">
         <div className="form"></div>
 
-        <form action=" " onSubmit={handleSubmit((data) => postData(data))}>
-          <h1>SIGN UP!</h1>
-          <input type="text" {...register('name')} value={JSON.parse(profileData).givenName} placeholder={JSON.parse(profileData).givenName} readOnly />
-          <input type="text" {...register('email')} value={JSON.parse(profileData).email} placeholder={JSON.parse(profileData).email} readOnly />
-          <input type="text" {...register('phone_number', { valueAsNumber: true })} placeholder="Whatsapp Number" pattern="^[6-9]\d{9}$" title="A valid number is required" required />
-          <input type="text" list="all_colleges" {...register('college_name')} placeholder="College/Institute" name="college_name" required />
-          <Collegelist id="all_colleges" />
-          <select id="years" className="givebgcolor" {...register('year')} required>
-            <option id="select-heading" value="Choose Year" disabled selected hidden>
-              Year
-            </option>
-            <option value="FIRST">First</option>
-            <option value="SECOND">Second</option>
-            <option value="THIRD">Third</option>
-            <option value="FOURTH">Fourth</option>
-            <option value="FIFTH">Fifth</option>
-          </select>
-          <input type="text" {...register('referral')} placeholder="Referral" />
-          <button type="submit" className="form-submit">
-            <span>SIGN UP</span>
-          </button>
-        </form>
+        {props.userEditing === true ? (
+          <form action=" " onSubmit={handleSubmit((data) => postData(data))}>
+            <h1>Update User!</h1>
+            <input type="text" {...register('name')} value={JSON.parse(profileData).name} placeholder={JSON.parse(profileData).name} readOnly />
+            <input type="text" {...register('email')} value={JSON.parse(profileData).email} placeholder={JSON.parse(profileData).email} readOnly />
+            <input type="text" {...register('phone_number', { valueAsNumber: true })} placeholder={JSON.parse(profileData).phone} pattern="^[6-9]\d{9}$" title="A valid number is required" required />
```

Comment:
add defaultValue={JSON.parse(profileData).college}
for college year and mobile number